package.json

```diff
@@ -58,9 +58,9 @@
   "devDependencies": {
     "@babel/core": "^7.4.5",
     "@babel/traverse": "^7.5.5",
+    "@storybook/addon-actions": "^5.1.10",
```

Comment:
Are this and the `package-lock` changes on purpose?

Reply:
I saw this and said "oh yeah someone modified package.json without using the CLI, that's ok." I didn't realize I wasn't actually installing `classnames` like I thought :( Thanks for catching this!

Reply:
Removing all changes to `package.json` and `package-lock.json` because they were handled in #70